.env.development

```diff
@@ -11,7 +11,7 @@ DATABASE_WITH_PHI_PASSWORD=
 DATABASE_WITH_PHI_USERNAME=
 
 RAILS_CACHE_REDIS_PASSWORD=
-RAILS_CACHE_REDIS_URL=localhost
+RAILS_CACHE_REDIS_URL="redis://localhost:6379/0"
```

Comment:
This is the dev config change needed for the updated, built in Redis caching support.  The deployment changes are the same, but just w/ the ElasticCache URLs instead.